### ISSUE TITLE
Control cells also analyzed for CNVs?

### ISSUE THREAD

**Issue by rhlgyb** [akdess/CaSpER#54]
Hi, I've been trying to use CaSpER for my data for a while now. I appreciate the many tutorials and sample data sets available.

One problem I am having is understanding the purpose of choosing control sample ids. (control cell names)
I believe the control cells are used as a template to compare the rest of the cells (I will call these cells the experimental cells from here on) to.
   
          If this is the case : 
          - for the BAF extract, should I use 1) only the BAM files of the experimental cells merged? or 2) BAM files of 
          control+experimental cells?

On my first attempt, I used the merged BAM file of control + experimental cells. The results for this indicated deletions or amplifications in not only the experimental cells but also the control cells.
I also tried the code for getting results without the BAF extract and only the expression values (as mentioned as an answer to one of the issues before) - results for this also returned cnv results for the control cells.

If I was correct, I wasn't expecting the control cells to also be detected to have CNVs, as they should have been deemed "normal" and have the experimental cells compared to them.

(the BAF extract also seems to be generated by comparing the BAM files to hg38 or hg19 and not the control cells...)

In short, what is the purpose for control cells and how/what are they used (for)?
Any clarifications or point-outs will really help me out.

Thank you!
beemilmlo

**Comment by laijen000** [comment]
Hi @beemilmlo , @akdess, 
I have the same question, and was wondering if you found a solution to this. I also have scRNA-seq data from controls and disease individuals, and would like to compare the burden of CNVs in the disease cells v. control cells. 

For generating the BAF file, should I merge the BAM files for the disease and control cells? Or should I generate a BAF file for each person separately? Thank you for your help!

**Comment by GuiSeSanz** [comment]
Hi everyone!

Have you been able to figure out how to perform correctly the analysis without generating the CNVs for the controls?
It is better to merge the control and the experimental cells?

Thank you for your help!

**Comment by lifan18** [comment]
Hi friends!

Thank you all for your recordings! It helps me to understand the questions.

I also confused at this issue. I cannot pass the stage of HMM segmentation for experimental groups BAF results (loh) and gene matrix with control samples.

Here are the errors: 
`
Performing recursive median filtering...
Performing HMM segmentation...
Processing cnv.scale:1 loh.scale:1...
Error in value[[jvseq[[jjj]]]] : subscript out of bounds
Calls: runCaSpER ... calculateLOHShiftsForEachSegment -> [<- -> [<-.data.frame
In addition: There were 50 or more warnings (use warnings() to see the first 50)
Execution halted
`

Did you find a solution already?

Best
